static/css/master.css

```diff
@@ -6063,3 +6062,15 @@ ul#myreads-paginator li.selected:hover {
 a.btn {
     text-decoration: none;
 }
+
+.generic-msg {
```

Comment:
SUGGESTION: Use a more meaningful selector name.

Reply:
I think this can actually be removed! Thanks!